psutil/__init__.py

```diff
@@ -47,6 +47,7 @@
 from ._compat import long
 from ._compat import PermissionError
 from ._compat import ProcessLookupError
+from ._compat import SubprocessTimeoutExpired
```

Comment:
Sorry, on a second thought actually I prefer `from ._compat import SubprocessTimeoutExpired as _SubprocessTimeoutExpired` (let's keep it private in the main psutil namespace). Can you update it?

Reply:
Updated properly now, sorry for the noise.